src/storage/buffer_manager/buffer_manager.cpp

```diff
@@ -469,7 +455,6 @@ void BufferManager::cachePageIntoFrame(FileHandle& fileHandle, page_idx_t pageId
 }
 
 void BufferManager::removeFilePagesFromFrames(FileHandle& fileHandle) {
-    evictionQueue.removeCandidatesForFile(fileHandle.getFileIndex());
```

Comment:
@royi-luo @benjaminwinger please double check the change here. I don't think we need to remove for file here, as we would `clearEvictedBMEntriesIfNeeded` at the end of checkpoint writes.